addons/hr/tests/test_hr_employee.py

```diff
@@ -7,6 +7,20 @@
 
 class TestHrEmployee(TestHrCommon):
 
+    def setUp(self):
+        super(TestHrEmployee, self).setUp()
```

Comment:
`super().setUp()` short syntax can be used when it gives the default param (current class, self).